cbv/models.py

```diff
@@ -312,11 +312,11 @@ def basic_yuml_data(self, first: bool = False) -> list[str]:
                     parent=ancestor.name,
                     child=self.name,
                     parent_col="white" if ancestor.is_secondary() else "lightblue",
-                    child_col="green"
-                    if first
-                    else "white"
-                    if self.is_secondary()
-                    else "lightblue",
+                    child_col=(
+                        "green"
+                        if first
+                        else "white" if self.is_secondary() else "lightblue"
+                    ),
```

Comment:
Note: We should look to refactor this line soon. (And ideally remove this display logic from the model entirely.)